### PR TITLE
FUI-391 - Allowed select to deal with isRequired being a string

### PR DESCRIPTION
**Pull request by TomFullalove**

### CODE REVIEW

File: js/components/utils/DataUtils.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * @description Some utility functions that can be shared across UI components
+ */
+
+/* eslint-disable import/prefer-default-export */
+
+export const checkBooleanString: any = (inputString) => typeof inputString === "boolean" ? inputString :
```
Comment (jamesbrattBoomi) [inline js/components/utils/DataUtils.ts]: Should the inputString argument not be typecast as a (string | boolean)?

File: js/components/select.tsx [hunk 2]
```diff
@@ -373,10 +374,12 @@ class Select extends React.Component<IItemsComponentProps, IDropDownState> {
             widthClassName = 'width-specified';
         }
 
+        const test = typeof model.isRequired;
```
Comment (jamesbrattBoomi) [inline js/components/select.tsx]: This needs removing

File: js/components/utils/DataUtils.ts (hunk 1 above)
Comment (TomFullalove) [reply js/components/utils/DataUtils.ts]: Yeah, I thought about doing that, but went instead for just returning false if the function were given anything but a string or a boolean.
I think it's probably best to add types though as you suggest, seems bad just to return false for invalid types.

File: js/components/select.tsx (hunk 2 above)
Comment (TomFullalove) [reply js/components/select.tsx]: oh whoops, sorry about that one

**Pull request merged**